docs/building.rst

```diff
@@ -10,17 +10,17 @@ Building ctags
 Build system add possibility to change program name
 ---------------------------------------------------------------------
 
-As on some systems (e.g. BSD) there is a 'ctags' program in the base
-system it's somewhat inconvenient to have the same name for universal-ctags
-During ``configure`` you can now change the output executable name.
+On some systems, like certain BSDs, there is already a 'ctags' program in the base
+system, so it is somewhat inconvenient to have the same name for 
+universal-ctags. During the ``configure`` stage you can now change 
+the output executable name.
 
-Add a prefix 'ex' which will result in 'ctags' transformed into 'exctags'
+To add a prefix 'ex' which will result in 'ctags' being transformed into 'exctags'
```

Comment:
I think the "which" would benefit from a preceding comma. Typically, when no comma is used the word "that" is used instead.